package/kernel/linux/modules/netfilter.mk

```diff
@@ -1148,21 +1148,44 @@ endef
 $(eval $(call KernelPackage,nft-nat))
 
 
+define KernelPackage/nft-offload-ipv4
+  SUBMENU:=$(NF_MENU)
+  TITLE:=Netfilter nf_tables routing/NAT offload support (ipv4)
+  DEPENDS:=!@LINUX_6_1 +kmod-nf-flow +kmod-nft-nat
+  KCONFIG:= CONFIG_NF_FLOW_TABLE_IPV6@lt5.17
+  FILES:= $(LINUX_DIR)/net/ipv4/netfilter/nf_flow_table_ipv4.ko@lt5.17
+  HIDDEN:=y
+  AUTOLOAD:=$(call AutoProbe,nf_flow_table_ipv4)
+endef
+
+$(eval $(call KernelPackage,nft-offload-ipv4))
+
+
+define KernelPackage/nft-offload-ipv6
+  SUBMENU:=$(NF_MENU)
+  TITLE:=Netfilter nf_tables routing/NAT offload support (ipv6)
+  DEPENDS:=!@LINUX_6_1 @IPV6 +kmod-nf-flow +kmod-nft-nat
```

Comment:
I think the dependency invert needs to be the other way around: `@!`.

Also, `-ipv4` Kconfigs IPV6